packages/gatsby/src/redux/actions.js

```diff
@@ -115,11 +104,75 @@ actions.createNode = (node, plugin = ``) => {
     return { type: `VALIDATION_ERROR`, error: true }
   }
 
+  // Ensure node isn't directly setting pluginFields.
+  if (node.pluginFields) {
+    throw new Error(
+      `Plugins creating nodes can not set data on the reserved field "pluginFields"
```

Comment:
I remember reading somewhere that [common-tags](https://github.com/declandewet/common-tags) were used. Would be nice to use `stripIndent` here, so that indentation can follow the code.